src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ActiveDirectoryTests.ps1

```diff
@@ -694,36 +698,82 @@ function Test-CreateDeleteAppPasswordCredentials
     Assert-AreEqual $cred2.Count 2
     $credCount = $cred2 | where {$_.KeyId -in $cred1.KeyId, $cred.KeyId}
     Assert-AreEqual $credCount.Count 2
+	$cred2 = $cred
+
+	# Add 1 key credential to the same app
+	$certCreated = 0
+	$cert = New-SelfSignedCertificate 
```

Comment:
You won't be able to use this for test runs in netcore - you should mark this test with the RunType=DesktopOnly trait